src/components/jellyfinActions.ts

```diff
@@ -381,11 +381,21 @@ export function play($scope: GlobalScope): void {
  * Don't actually stop, just show the idle view after 20ms
  */
 export function stop(): void {
-    setTimeout(function () {
+    setTimeout(() => {
         setAppStatus('waiting');
     }, 20);
 }
 
+/**
+ * @param item
+ * @param maxBitrate
+ * @param deviceProfile
+ * @param startPosition
+ * @param mediaSourceId
+ * @param audioStreamIndex
+ * @param subtitleStreamIndex
+ * @param liveStreamId
+ */
```

Comment:
Don't check this in without completing the JSDoc please, it's not good to add this just to squelsh the lint error/warning, also the one just below.

Reply:
There should now be a lint rule to warn on empty blocks as well, so this should still produce a warning